lib/code_corps/emails/message_for_project_email.ex

```diff
@@ -0,0 +1,43 @@
+defmodule CodeCorps.Emails.MessageForProjectEmail do
+  import Bamboo.Email, only: [to: 2]
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.{
+    Conversation,
+    Emails.BaseEmail,
+    Message,
+    Organization,
+    Project,
+    User,
+    WebClient
+  }
+
+  @spec create(Message.t, Conversation.t, User.t) :: Bamboo.Email.t
+  def create(
+    %Message{project: %Project{} = project},
+    %Conversation{} = conversation,
+    %User{} = user) do
+
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), %{
+      conversation_url: project |> conversation_url(conversation),
+      name: user.first_name,
+      project_title: project.title,
+      subject: "#{user.first_name} has posted a reply on #{project.title}"
```

Comment:
Different from the user email primarily because this could be a reply by any participant, including another admin/owner

Reply:
I might change some language here but need to think on it.

Reply:
Maybe:

> "#{user.first_name} replied to a message in #{project.title}"